kivy/uix/textinput.py

```diff
@@ -454,7 +454,7 @@ class TextInput(FocusBehavior, Widget):
                   'on_quad_touch')
 
     def __init__(self, **kwargs):
-        self.always_release_bubble = kwargs.get('always_release_bubble', False)
+        self.always_release_bubble = kwargs.get('always_release_bubble', True)
```

Comment:
I think this is just hiding the behavior change, but we are still able to see it. 😁  Do not modify `always_release` in any form.